Source/Core/Core/ConfigManager.h

```diff
@@ -73,9 +73,15 @@ struct SConfig
   void SetRunningGameMetadata(const DiscIO::Volume& volume, const DiscIO::Partition& partition);
   void SetRunningGameMetadata(const IOS::ES::TMDReader& tmd, DiscIO::Platform platform);
   void SetRunningGameMetadata(const std::string& game_id);
-  // Reloads title-specific map files, patches, custom textures, etc.
-  // This should only be called after the new title has been loaded into memory.
-  static void OnNewTitleLoad(const Core::CPUThreadGuard& guard);
+
+  // Triggered when Dolphin loads a title directly
+  // Reloads title-specific map files, patches, etc.
+  static void OnTitleBooted(const Core::CPUThreadGuard& guard);
```

Comment:
I don't think this rename is much better. These three callbacks are all so similar. Maybe `OnTitleDirectlyBooted` if that's what it is?

Reply:
Yeah!